packages/starrating/src/react/star.js

```diff
@@ -7,7 +7,7 @@ import { withTheme } from '@pluralsight/ps-design-system-theme/react'
 
 import css, { BASE_CLASSNAME } from '../css'
 
-const APPEARANCES = {
+export const APPEARANCES = {
```

Comment:
I'm not seeing where these are used outside this module.

Reply:
oops - will remove